releasetasks/templates/enUS.yml.tmpl

```diff
@@ -48,6 +48,13 @@
             {% if running_tests is defined %}
             task_name: "{{ complete_beetmover_basename }}"
             {% endif %}
+            build_props:
+                locales: ["en-US"]
+                branch: {{ branch }}
+                platform: {{ platform }}
+                version: "{{ version }}"
```

Comment:
should we stick product in here now to prep for upcoming fennec or will we have logic against platform?